tests/test_command_generation.py

```diff
@@ -0,0 +1,347 @@
+# BSD 2-Clause License
```

Comment:
It seems like your branch has the old test directory structure, can you pull in the changes from smartsim-refactor so the test directory is up to date and rerun your new tests to validate that they pass?

Reply:
Will do!